R/graticule.R

```diff
@@ -183,19 +188,21 @@ graticule_attributes = function(df) {
 		function(x) { y = x[[length(x)]]; n = nrow(y); apply(y[(n-1):n,], 2, diff) } ))
 	df$angle_end = apply(dxdy, 1, function(x) atan2(x[2], x[1])*180/pi)
 	bb = st_bbox(df)
-    selE = df$type == "E" & df$y_start < min(df$y_start) + 0.001 * (bb[3] - bb[1])
+  selE = df$type == "E" & df$y_start < min(df$y_start) + 0.001 * (bb[3] - bb[1])
 	selN = df$type == "N" & df$x_start < min(df$x_start) + 0.001 * (bb[4] - bb[2])
 	df$plot12 = selE | selN
 	df
 }
 
-trim_bb = function(bb = c(-180, -90, 180, 90), margin) {
+trim_bb = function(bb = c(-180, -90, 180, 90), margin, wrap=c(-180,180)) {
```

Comment:
Borrowing `wrap`=c(-180,180) default argument from [maps::map()](https://www.rdocumentation.org/packages/maps/versions/3.2.0/topics/map):

- **wrap**:  Boolean or a numeric vector. If TRUE, lines that cross too far across the map (due to a strange projection) are omitted. If wrap is a vector of length 2 or more, it is interpreted as the longitude range to be used for a global map, e.g. c(-180,180) or c(0,360). This wrapping even works when fill=TRUE and is performed before any projection (so the range must always be in degrees). However, the wrapping is performed after xlim, ylim are applied, so these options should probably never be combined. If there is a third component, this signifies the latitude at which Antarctica will be "closed". The default value is -89.9. Special values are NA (don't draw Antarctica at all) and 0 (draw the line at the latitude of the extremal points, not at a fixed lower latitude).